refm/api/src/_builtin/Integer

```diff
@@ -867,27 +867,66 @@ self の絶対値を返します。
   2 ^ 3 # => 1
 
 --- [](nth) -> Integer
+#@since 2.6.0
+--- [](nth, len) -> Integer
+#@end
+#@since 2.7.0
+--- [](range) -> Integer
+#@end
 
 nth 番目のビット(最下位ビット(LSB)が 0 番目)が立っている時 1
 を、そうでなければ 0 を返します。
 
-@param nth  何ビット目を指すかの数値
+@param nth   何ビット目を指すかの数値
+#@since 2.6.0
```

Comment:
2.6時点では第2引数なさそう?
```
 docker run --rm -it rubylang/all-ruby
root@1b5ca604348c:/all-ruby# ./all-ruby
usage: all-ruby RUBY-ARGS
environment variables:
  ALL_RUBY_SINCE=ruby-1.4               run only versions since specified one
  ALL_RUBY_BINS='ruby-2.4.4 ruby-2.5.1' run only versions specfied
  ALL_RUBY_ADDBINS=./ruby               run specified commands additionaly
  ALL_RUBY_SHOW_DUP=yes                 don't suppress duplicated output
root@1b5ca604348c:/all-ruby# ALL_RUBY_SINCE=ruby-2.5 ./all-ruby -e '255[1,1]'
ruby-2.5.0          -e:1:in `[]': wrong number of arguments (given 2, expected 1) (ArgumentError)
                        from -e:1:in `<main>'
                exit 1
...
ruby-2.6.5          -e:1:in `[]': wrong number of arguments (given 2, expected 1) (ArgumentError)
                        from -e:1:in `<main>'
                exit 1
ruby-2.7.0-preview1
...
ruby-2.7.0
```

Reply:
メモが残っていないのでよくわからないのですが、何か確認方法が間違っていたようでした。
再度確認してみたら 2.7 からの変更のみだったので、全体的に 2.7 からの分岐のみにしました。